arroyo-rpc/proto/rpc.proto

```diff
@@ -317,6 +317,18 @@ service WorkerGrpc {
   rpc JobFinished(JobFinishedReq) returns (JobFinishedResp);
 }
 
+// Scheduler
+
+message StartPipelineReq {
+  string name = 1;
+  string pipeline_path = 2;
+  string wasm_path = 3;
+  string job_id = 4;
+  string hash = 5;
+  int64 run_id = 6;
+  uint64 slots = 7;
+  map<string, string> environment_variables = 8;
```

Comment:
nit: we tend to shorten this to env_vars everywhere else in our code

Reply:
Will do.